repos/fdbt-site/tests/pages/api/returnService.test.ts

```diff
@@ -0,0 +1,42 @@
+import returnService from '../../../src/pages/api/returnService';
+import { getMockRequestAndResponse, mockRawService } from '../../testData/mockData';
+import * as auroradb from '../../../src/data/auroradb';
+import { TXC_SOURCE_ATTRIBUTE } from '../../../src/constants/attributes';
+
+beforeEach(() => {
+    jest.resetAllMocks();
+
+    jest.spyOn(auroradb, 'getServiceByIdAndDataSource').mockResolvedValue(mockRawService);
+});
+
+describe('service', () => {
+    it('should return 302 redirect to /returnService when there is no body in the request', async () => {
```

Comment:
And it should set errors on the session object, we need to include that in the test.

Reply:
not getting this point?